Sources/Sentry/Public/SentryOptions.h

```diff
@@ -8,6 +8,12 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(Options)
 @interface SentryOptions : NSObject
 
+/**
+ * Optional, custom cache directory. Use when default one can not be accessed, e.g. in security
+ * environment.
```

Comment:
`m`: A bit clearer, IMO.
```suggestion
 * The path to store SDK data, like events, transactions, profiles, raw crash data, etc. We
 recommend only changing this when the default, e.g., in security environments, can't be accessed.
 * 
 * @note The default is `NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask,
 YES)`.
```